code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -1380,6 +1382,43 @@
 	burst_shot_delay = 2.2
 	spread = 18
 
+/obj/item/gun/ballistic/automatic/assault_carbine/olympic
+	name = "OA-93 machine pistol"
+	desc = "A pistol-sized Colt assault carbine chambered for 5mm made by Olympic Arms, issued to Vertibird pilots. It's lighter, concealable and boasts a higher rate of fire - though the short barrel and lack of a buttstock make it a very poor choice for anything but boarding encounters."
+	icon_state = "olympic"
+	item_state = "assault_carbine"
+	icon_prefix = "olympic"
+	icon = 'icons/fallout/objects/guns/ballistic.dmi'
+	slowdown = 0.15
+	mag_type = /obj/item/ammo_box/magazine/m5mm
+	autofire_shot_delay = 0.9
```

Comment:
Way too fast in general

Reply:
With the 30 spread, it's effectively a bullet hose. So anything beyond 5 tiles is a no-go zone.